### ISSUE TITLE
Throw an Exception if provided a molecule with elements that are not covered

### ISSUE THREAD

**Issue by jchodera** [choderalab/espaloma#190]
We need to be sure to throw an `Exception` if provided a molecule that contains elements that are not covered.

**Comment by mikemhenry** [comment]
Good idea, we should probably make a subclass of `Exception` like `ElementParameterizationError` so users can handle the exception if they want to.
